pkg/env/utils.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 The Okteto Authors
```

Comment:
Nit: not blocker

We could try to find another name for the file. It's our opportunity to avoid generic names la utils, common, etc...

We could call it parse, load, value. I prefer parse as it is how it is call the main function to read the value

Reply:
Agree. I think it could even fit in the `env` file given that we are expanding env vars there. WDYT?

Reply:
I agree on moving it to `env.go` for now: 2a660aa4